src/components/forms/Fields.tsx

```diff
@@ -46,10 +46,12 @@ export const InvalidFieldHints = ({
   formStatus: string;
   className?: string;
 }) => {
-  const { INVALID_FIELDS, INVALID_REQUIRED_FIELDS, INVALID_EMAIL, FETCH_ERROR } = FORM_STATUSES;
+  const { INVALID_FIELDS, INVALID_SIZE, INVALID_REQUIRED_FIELDS, INVALID_EMAIL, FETCH_ERROR } =
+    FORM_STATUSES;
   return (
     <small className={`text-danger form-error-message ${className}`}>
       {formStatus === INVALID_FIELDS && <Trans>Please fill in all fields</Trans>}
+      {formStatus === INVALID_SIZE && <Trans>Please fill in a company size bigger than 0</Trans>}
```

Comment:
Since it also applies to stakeholders
```suggestion
      {formStatus === INVALID_SIZE && <Trans>Please enter a number larger than 0</Trans>}
```